plugin-server/src/main/pluginsServer.ts

```diff
@@ -292,7 +292,9 @@ export async function startPluginsServer(
             })
         }
 
-        if (hub.capabilities.ingestionOverflow) {
+        if (hub.capabilities.ingestionOverflow && !hub.capabilities.ingestion) {
+            // ingestionOverflow may be enabled with ingestion to signal we are producing ingestion events
+            // to the OVERFLOW topic. But we don't want to start an additional consumer.
```

Comment:
I think we always want to start the ingestion overflow consumer if `ingestionOverflow` is true.

Reply:
It depends, if we are also using it as a flag to indicate events should be re-routed, then we don't as we would be starting two consumers in the ingestion deployment.

Reply:
Made this commit to show how we could do this, though, as an alternative to a new env variable. But not saying it has to stay like this.

Reply:
Right, I was thinking capabilities would have a simple relationship to what gets started.

As it stands in the ingestion pod, `hub.capabilities.ingestionOverflow` will always be false as we have PLUGIN_SERVER_MODE="ingestion", so in this case we'd change the capabilities logic to also check something other than PLUGIN_SERVER_MODE?